.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started
+
+Thank you for your interest in contributing to the Parse.ly plugin!  We hope this document helps you get set up with everything you need to contribute and we look forward to working with you!
+
+## Reporting Bugs
+
+Please search the [repo's issues](https://github.com/Parsely/wp-parsely/issues) to see if your issue has been reported already and if so, comment on that issue instead of opening a new one.
+
+When creating a new issue, please add specific steps to reproduce the problem, upload any relevant screenshots, and describe what happened and what you expected would happen instead.
+
+## Setting up your environment
+
+### Minimum required versions
+
+* #### Node.js - LTS
```

Comment:
Currently, [the plugin requires 14, and NPM 7](https://github.com/Parsely/wp-parsely/blob/514fc01bdcde57987c1166f4c622fd53ced17f2e/package.json#L7-L8).